ui/core/src/assets.ethereum.mainnet.json

```diff
@@ -1,418 +1,514 @@
+    "decimals": 18,
+    "imageUrl": "https://assets.coingecko.com/coins/images/877/small/chainlink-new-logo.png?1547034700",
+    "name": "Chainlink",
+    "network": "ethreum",
+    "price_usd": 21.78,
+    "symbol": "LINK",
+    "transferLimit": 229568411386593180000
+  },
+  {
+    "address": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+    "decimals": 6,
+    "imageUrl": "https://assets.coingecko.com/coins/images/6319/small/USD_Coin_icon.png?1547042389",
+    "name": "USD Coin",
+    "network": "ethreum",
+    "price_usd": 0.998051,
+    "symbol": "USDC",
+    "transferLimit": 5009764030
+  },
+  {
+    "address": "0x6f259637dcd74c767781e37bc6133cd6a68aa161",
+    "decimals": 18,
+    "imageUrl": "https://assets.coingecko.com/coins/images/2822/small/huobi-token-logo.png?1547036992",
+    "name": "Huobi Token",
+    "network": "ethreum",
+    "price_usd": 5.44,
+    "symbol": "HT",
+    "transferLimit": 919117647058823400000
+  },
+  {
+    "address": "0xc00e94cb662c3520282e6f5717214004a7f26888",
+    "decimals": 18,
+    "imageUrl": "https://assets.coingecko.com/coins/images/10775/small/COMP.png?1592625425",
+    "name": "Compound",
+    "network": "ethreum",
+    "price_usd": 225.92,
+    "symbol": "COMP",
+    "transferLimit": 22131728045325780000
+  },
+  {
+    "address": "0x9f8f72aa9304c8b593d555f12ef6589cc3a579a2",
+    "decimals": 18,
+    "imageUrl": "https://assets.coingecko.com/coins/images/1364/small/Mark_Maker.png?1585191826",
+    "name": "Maker",
+    "network": "ethreum",
+    "price_usd": 1452.52,
+    "symbol": "MKR",
+    "transferLimit": 3442293393550519300
+  },
+  {
+    "address": "0xf1290473e210b2108a85237fbcd7b6eb42cc654f",
+    "decimals": 18,
+    "imageUrl": "https://assets.coingecko.com/coins/images/7196/small/to3Vj4EZ_400x400.jpg?1547043758",
+    "name": "HedgeTrade",
+    "network": "ethreum",
+    "price_usd": 1.89,
+    "symbol": "HEDG",
+    "transferLimit": 2.6455026455026456e+21
+  },
+  {
+    "address": "0x0d8775f648430679a709e98d2b0cb6250d2887ef",
+    "decimals": 18,
+    "imageUrl": "https://assets.coingecko.com/coins/images/677/small/basic-attention-token.png?1547034427",
+    "name": "Basic Attention Token",
+    "network": "ethreum",
+    "price_usd": 0.287687,
+    "symbol": "BAT",
+    "transferLimit": 1.737999979144e+22
+  },
+  {
+    "address": "0x75231f58b43240c9718dd58b4967c5114342a86c",
+    "decimals": 18,
+    "imageUrl": "https://assets.coingecko.com/coins/images/4463/small/okb_token.png?1548386209",
+    "name": "OKB",
+    "network": "ethreum",
+    "price_usd": 6.51,
+    "symbol": "OKB",
+    "transferLimit": 768049155145929300000
+  },
+  {
+    "address": "0x8e870d67f660d95d5be530380d0ec0bd388289e1",
+    "decimals": 18,
+    "imageUrl": "https://assets.coingecko.com/coins/images/6013/small/paxos_standard.png?1548386291",
+    "name": "Paxos Standard",
+    "network": "ethreum",
+    "price_usd": 0.999397,
+    "symbol": "PAX",
+    "transferLimit": 5.003016819141942e+21
```

Comment:
This needs to be turned into a string, and not have e+21 in it. Same goes for all other tokens that look like this.